common/errors_go13_test.go

```diff
@@ -43,6 +43,7 @@ func TestDownstreamError_Unwrap(t *testing.T) {
 	r.WriteHeader(http.StatusInternalServerError)
 
 	resp := r.Result()
+	defer resp.Body.Close()
```

Comment:
This change has been fixed as part of separate pr - https://github.com/anz-bank/sysl-go/pull/253